juju/client/connection.py

```diff
@@ -267,11 +273,13 @@ def https_connection(self):
         username = accounts['user']
         password = accounts.get('password')
         models = jujudata.models()[controller_name]
+        if '/' not in model_name:
+            model_name = '{}/{}'.format(username, model_name)
```

Comment:
this looks like it should be part of determining the model_name logic above. This could include a : ?